cmd/bridge/main.go

```diff
@@ -540,6 +585,39 @@ func main() {
 		if srv.Auther, err = auth.NewAuthenticator(context.Background(), oidcClientConfig); err != nil {
 			klog.Fatalf("Error initializing authenticator: %v", err)
 		}
+
+		srv.Authers = make(map[string]*auth.Authenticator)
+		srv.Authers["local-cluster"] = srv.Auther
+
+		if len(managedClusterConfigs) > 0 {
+			for _, managedCluster := range managedClusterConfigs {
+				managedClusterOIDCClientConfig := &auth.Config{
+					AuthSource:   authSource,
+					IssuerURL:    managedCluster.Server,
+					IssuerCA:     *fUserAuthOIDCCAFile,
+					ClientID:     *fUserAuthOIDCClientID,
+					ClientSecret: oidcClientSecret,
+					RedirectURL:  proxy.SingleJoiningSlash(srv.BaseURL.String(), fmt.Sprintf("%s/%s", server.AuthLoginCallbackEndpoint, managedCluster.Name)),
+					Scope:        scopes,
+
+					// Use the k8s CA file for OpenShift OAuth metadata discovery.
+					// This might be different than IssuerCA.
+					K8sCA: caCertFilePath,
+
+					ErrorURL:   authLoginErrorEndpoint,
+					SuccessURL: authLoginSuccessEndpoint,
+
+					CookiePath:    cookiePath,
+					RefererPath:   refererPath,
+					SecureCookies: secureCookies,
+					ClusterName:   managedCluster.Name,
+				}
+
+				if srv.Authers[managedCluster.Name], err = auth.NewAuthenticator(context.Background(), managedClusterOIDCClientConfig); err != nil {
+					klog.Fatalf("Error initializing managed cluster authenticator: %v", err)
+				}
+			}
+		}
```

Comment:
@spadgett I assume this is going to have to change, but I'm not really sure where we landed on what we are doing about auth.